src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs

```diff
@@ -87,15 +87,15 @@
                      (nav/navigate (:nav-context props) "workspaces" (common/row->workspace-id data))
                      (comps/push-message {:header "Request Access"
                                           :message
-                                            (if (= (config/tcga-namespace) (:namespace data))
-                                             [:span {}
-                                               [:p {} "For access to TCGA protected data please apply for access via dbGaP [instructions can be found "
-                                               [:a {:href "https://wiki.nci.nih.gov/display/TCGA/Application+Process" :target "_blank"} "here"] "]." ]
-                                               [:p {} "After dbGaP approves your application please link your eRA Commons ID in your FireCloud profile page."]]
-                                             [:span {}
+                                          (if (= (config/tcga-namespace) (:namespace data))
+                                            [:span {}
+                                             [:p {} "For access to TCGA protected data please apply for access via dbGaP [instructions can be found "
+                                              [:a {:href "https://wiki.nci.nih.gov/display/TCGA/Application+Process" :target "_blank"} "here"] "]." ]
+                                             [:p {} "After dbGaP approves your application please link your eRA Commons ID in your FireCloud profile page."]]
+                                            [:span {}
                                              "Please contact " [:a {:target "_blank" :href (str "mailto:" (:library:contactEmail data))} (str (:library:datasetCustodian data) " <" (:library:contactEmail data) ">")]
-                                               " and request access for the "
-                                               (:namespace data) "/" (:name data) " workspace."])})))}))
+                                             " and request access for the "
+                                             (:namespace data) "/" (:name data) " workspace."])})))}))
```

Comment:
I did the intellij auto-indent lines (I thought I had selected just my stuff but apparently not)so I *think* this is right